src/scripts/clipperUI/clipper.tsx

```diff
@@ -576,41 +576,36 @@ class ClipperClass extends ComponentBase<ClipperState, {}> {
 		let handleSignInEvent = new Log.Event.PromiseEvent(Log.Event.Label.HandleSignInEvent);
 
 		this.setState({ userResult: { status: Status.InProgress } });
-		type ErrorObject = { correlationId?: string, error: string, errorDescription: string };
-		Clipper.getExtensionCommunicator().callRemoteFunction(Constants.FunctionKeys.signInUser, { param: authType, callback: (data: UserInfo | ErrorObject) => {
+		type ErrorObject = { updateReason: UpdateReason, correlationId?: string, error: string, errorDescription: string };
```

Comment:
nit: can we declare the type so it looks more like an object instead of all on one line